README.md

```diff
@@ -125,10 +125,10 @@ the cookie back to the server in the future if the browser does not have an HTTP
 connection.
 
 Please note that `secure: true` is a **recommended** option. However, it requires
-an https-enabled website, i.e., HTTPS is necessary for secure cookies. If `secure`
-is set, and you access your site over HTTP, the cookie will not be set. If you
-have your node.js behind a proxy and are using `secure: true`, you need to set
-"trust proxy" in express:
+an https-enabled website (or connection via localhost), i.e., HTTPS is necessary
+for secure cookies. If `secure` is set, and you access your site over HTTP, the 
```

Comment:
> HTTPS is necessary
for secure cookies

Probably this line would need reworded with a change like this, as I believe that is the whole point of this change, that HTTP would work for localhost.

Reply:
Take a look at 8314c37, I think it is clear that way

Reply:
Thanks. I left a comment already on it 👍